traits/trait_notifiers.py

```diff
@@ -361,7 +361,7 @@ def equals(self, handler):
         return False
 
 
-class StaticAnyTraitChangeNotifyWrapper(AbstractStaticChangeNotifyWrapper):
+class StaticAnytraitChangeNotifyWrapper(AbstractStaticChangeNotifyWrapper):
```

Comment:
This is technically a backwards-incompatible API change, right?  I think this is safe, since it is pretty deep internals and should only be used as part of the `anytrait` stuff.

Reply:
Agreed. I did a search for any external uses of this, and found none. It _is_ documented in the API docs, but it's not exposed in `traits.api`.